docs/main/user-guides/upgrades.md

```diff
@@ -58,27 +58,42 @@ helm get values openebs -n openebs -o yaml > old-values.yaml
 ```
 
 :::note
-If you are using a custom helm chart to use OpenEBS, compare your set of helm values against the new helm values and the values of the dependency helm charts, [here](https://github.com/openebs/openebs/blob/HEAD/charts/values.yaml).
+If you are using a custom helm chart to use OpenEBS, compare your set of helm values against the new helm values and the values of the dependency helm charts. Refer [values.yaml](https://github.com/openebs/openebs/blob/HEAD/charts/values.yaml) for more details.
 
-For users who have used helm v3.13 or above to install their chart, and not used helm's `--set` and/or `-f` options to configure their chart values, using the `-a` option with your `helm get values` command will let you capture your configuration values.
+If you have used helm v3.13 or above to install their chart, and not used helm's `--set` and/or `-f` options to configure their chart values, using the `-a` option with your `helm get values` command will let you capture your configuration values.
  
-Feel free to reach out via our communication channels.
+Feel free to reach out via our [communication channels](../community.md).
 :::
 
 
-## Local Storage
+## Local Storage Upgrade
 
-This section describes the Local Storage upgrade from OpenEBS chart 3.x to OpenEBS 4.1.1. The upgrade process for Local PV Hostpath, Local PV LVM, and Local PV ZFS are largely identical, with a few changes in helm values depending on the Local PV Storage variant we are upgrading from.
+The upgrade process for Local PV Hostpath, Local PV LVM, and Local PV ZFS are largely identical, with a few changes in helm values depending on the Local PV Storage variant we are upgrading from.
+
+### From 4.x to 4.1.1 or Higher
```

Comment:
Should it not be 4.2 here as well?

Reply:
We can upgrade from 4.x version to 4.1.1 version using these steps. So we kept the heading as "**From 4.x to 4.1.1 or Higher**" @avishnu

Reply:
Hmm. 4.x to 4.1.1 or higher is incorrect, because 4.x can imply 4.1 as well

Reply:
Updated the docs as per the discussion.